srcpkgs/opentoonz/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'opentoonz'
+pkgname=opentoonz
+version=1.3.0
+revision=1
+wrksrc="${pkgname}-${version}"
```

Comment:
This is the default, it can be omitted

Reply:
Removed. The only reason why `wrksrc` was set is that it didn't work in `configure_args`...